src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.scss

```diff
@@ -1,50 +1,145 @@
-.media {
-  display: block;
-  min-height: 60px;
-  width: 100%;
-
-  .media-left {
-    border-top-left-radius: 2px;
-    border-top-right-radius: 0;
-    border-bottom-right-radius: 0;
-    border-bottom-left-radius: 2px;
-    display: block;
-    float: left;
-    height: 60px;
-    width: 60px;
-    text-align: center;
-    font-size: 40px;
-    line-height: 60px;
-    padding-right: 0;
-
-    .fa {
-      font-size: 45px;
-    }
-  }
-
-  .media-body {
-    padding: 5px 10px;
-    margin-left: 60px;
-
-    .media-heading {
-      display: block;
-      font-size: 20px;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
-
-      .underline {
-        border-bottom: 1px solid #e5e5e5;
-        margin-top: 0px;
-        margin-bottom: 18px;
-        padding: 0px;
-      }
-    }
-
-    .media-text {
-      display: block;
-      font-weight: bold;
-      font-size: 18px;
+$card-height: 8vw;
+$card-medium-height: 13vw;
+
+.card {
+  border: 0.5px solid #ddd;
+  border-radius: 3px;
+  background: #f5f5f5;
+  box-shadow: 0 1px 1px rgba(3, 3, 3, 0.175);
+  margin: 0 -10px 20px;
+  padding: 0 20px;
+  width: auto;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+
+  min-height: $card-height;
+}
+
+.card-title {
+  font-size: 1.2vw;
+  margin: 20px 0;
+  padding: 0;
+}
+
+.card-body {
+  font-size: 1.25vw;
+  padding: 0 0 20px;
+  text-align: center;
+  padding-top: 2%;
+  padding-bottom: 2%;
+}
+
+@media (max-width: 600px) {
+  .card-body {
+    margin-top: -10%;
+  }
+}
+
+@media (min-width: 601px) {
+  .card-body {
+    margin-top: -5%;
+  }
+}
+
+@media (min-width: 992px) {
+  .card-body {
+    margin-top: -10%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .card-body {
+    margin-top: -2%;
+  }
+}
+
+@media (min-width: 1400px) {
+  .card-body {
+    margin-top: 0;
+  }
+}
+
+.card-link {
+  cursor: pointer;
+}
+
+.content-highlight {
+  font-weight: bold;
+}
+
+.content-medium {
+  font-size: 1.7vw;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.card-medium {
+  min-height: $card-medium-height;
+
+  .card-title {
+  }
+
+  @media (max-width: 600px) {
```

Comment:
You should put all css rules for this media query in the same block. No reason to declare 3 media queries for the same size.
Sames goes for the others.

Reply:
OK, although AFAIK the number of media queries does not have any performance impact. The separation seemed to me more readable.

Reply:
Done.